docs/source/getting-started/content/proc_troubleshooting-expired-certificates.adoc

```diff
@@ -0,0 +1,37 @@
+[id="troubleshooting-expired-certificates_{context}"]
+= Troubleshooting expired certificates
+
+Each released `{bin}` binary expires 30 days after the release due to certificates embedded in the OpenShift cluster.
+Using an older `{bin}` binary will result in an expired certificates error.
```

Comment:
This sentence is misleading, at the moment we force crc binaries to be used with a given bundle version, but it really is using the older *bundle* which has the certificate issues.
If it was possible to run latest binary with an older bundle, we'd still get certificate issues.

Reply:
Good point, will update soon. Thanks!